### DISCUSSION TITLE
Regarding the LMV_layout_v2 and LMV_Layout_v3

### DISCUSSION THREAD

**Discussion by amanflexday** [PaddlePaddle/PaddleOCR#12813] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

I want to know how paddle Ocr is licensed by Apache when Layout_LMV2 and Layout _LMV3 are not for commercial use ?

**Comment by amanflexday** [comment]
I want to know how paddle Ocr is licensed by Apache when Layout_LMV2 and Layout _LMV3 are not for commercial use ?

**Comment by SWHL** [comment]
Please point out where PaddleOCR uses the LMV_layout_v2
